diffsims/tests/test_crystallography/test_reciprocal_lattice_point.py

```diff
@@ -286,6 +286,26 @@ def test_calculate_theta(self, ferrite_phase, voltage, hkl, desired_theta):
         rlp.calculate_theta(voltage=voltage)
         assert np.allclose(rlp.theta, desired_theta)
 
+    @pytest.mark.parametrize(
+        "hkl, family_heads, family_idx, n_families",
+        [
+            (
+                [[1, 1, 1], [1, 0, 0], [-1, -1, -1], [0, 1, 0], [2, 0, 0]],
+                [[1, 1, 1], [1, 0, 0]],
```

Comment:
@hakonanes  my point is that if the system is hexagonal, tetragonal, monoclinic or triclinic - the correct answer here should be family_heads=[[1,1,1], [1,0,0], [0,1,0]] - I don't think that's currently tested for? Or what the code would return?

In response to your other question, it's quite possible that the current code only works for cubic systems... but always looking to improve, we should at least log it as an issue if that is the case...

Reply:
Oh, and I've just seen your update to your comment - I think we're all on the same page now :)

Reply:
The point group symmetry of the `ReciprocalLatticePoint` instance is only considered in the `.symmetrise()` method, not in the `.unique()` method, thus the logic in the latter method should only works for cubic crystals as it is implemented... We should fix this, see my comment below.